code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -251,15 +251,13 @@
 	L.SetStuttering(4 SECONDS)
 
 	ADD_TRAIT(L, TRAIT_WAS_BATONNED, user_UID) // so one person cannot hit the same person with two separate batons
-	L.apply_status_effect(STATUS_EFFECT_DELAYED, 2 SECONDS, CALLBACK(L, TYPE_PROC_REF(/mob/living, KnockDown), knockdown_duration), COMSIG_LIVING_CLEAR_STUNS)
 	addtimer(CALLBACK(src, PROC_REF(baton_delay), L, user_UID), 2 SECONDS)
 
 	SEND_SIGNAL(L, COMSIG_LIVING_MINOR_SHOCK, 33)
 
 	L.lastattacker = user.real_name
 	L.lastattackerckey = user.ckey
-	L.visible_message("<span class='danger'>[user] has stunned [L] with [src]!</span>",
-		"<span class='userdanger'>[L == user ? "You stun yourself" : "[user] has stunned you"] with [src]!</span>")
+	L.visible_message("<span class='danger'>[src] has stunned [L]!</span>")
```

Comment:
Code is fine, but I'm trying to think of a better way of this sounding smooth when all 3 of these lines are read, this is how it is currently. The problem is it switches between past tense and present tense. I think maybe this visiuble message on this line should be in present tense, its just to me "has stunned" sounds a bit awkward.
```
...has thrown...
*has stunned*
...is hit by...
```